### DISCUSSION TITLE
Move Over Existing Postgres Database to Supabase

### DISCUSSION THREAD

**Discussion by osseonews** [supabase/supabase#4168] [Questions] [answered]
Looked everywhere in the docs for this, and am curious, how would we move an existing Postgres database to supabase? Can we do a pg_dump on the existing database and then just import into supabase via a restore? Can we just connect to supabase via pgadmin to facilitate the above?

**Comment by soedirgo** [comment]
We don't have a "well-trodden path" migration strategy, but you can try doing it similar to [this guide](https://supabase.com/docs/guides/database#migrating-between-projects). If all your db objects are in the same schema, you can try limiting the `pg_dump` to that schema to minimize errors.

**Comment by J0** [comment, marked as answer]
We now have a set of [migration guides](https://supabase.com/docs/guides/migrations/heroku) that can be referred to. But as @soedirgo has mentioned `pg_dump` is also a viable option if  all your db objects are in the same schema.
